runners/core-java/src/test/java/org/apache/beam/runners/core/StateInternalsTest.java

```diff
@@ -55,27 +55,27 @@
  */
 public abstract class StateInternalsTest {
 
-  private static final BoundedWindow WINDOW_1 = new IntervalWindow(new Instant(0), new Instant(10));
-  static final StateNamespace NAMESPACE_1 = new StateNamespaceForTest("ns1");
-  private static final StateNamespace NAMESPACE_2 = new StateNamespaceForTest("ns2");
-  private static final StateNamespace NAMESPACE_3 = new StateNamespaceForTest("ns3");
+  public static final BoundedWindow WINDOW_1 = new IntervalWindow(new Instant(0), new Instant(10));
+  public static final StateNamespace NAMESPACE_1 = new StateNamespaceForTest("ns1");
+  public static final StateNamespace NAMESPACE_2 = new StateNamespaceForTest("ns2");
+  public static final StateNamespace NAMESPACE_3 = new StateNamespaceForTest("ns3");
 
-  static final StateTag<ValueState<String>> STRING_VALUE_ADDR =
+  public static final StateTag<ValueState<String>> STRING_VALUE_ADDR =
```

Comment:
I think these should stay private. Commented where I saw one used. You can just make fake namespaces for other suites, to keep them independent.